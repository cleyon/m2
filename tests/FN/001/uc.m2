@define lower Hey You!!  yes, I'm talking to you...
@uc lower@
