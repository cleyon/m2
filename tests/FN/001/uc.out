HEY YOU!!  YES, I'M TALKING TO YOU...
